packages/fxa-payments-server/src/components/AlertBar/index.tsx

```diff
@@ -24,48 +25,81 @@ export const AlertBar = ({
   actionButton,
   checked,
   children,
-  className = 'alert',
+  className,
```

Comment:
Would it make sense to rename this parameter to something like `type`, since we aren't using it to pass in additional class names anymore? Changing the name would make it's intent a bit more clear.

Reply:
@xlisachan we should be able to make the change now, since we aren't using `className` in the `AlertBar` component, except to decide which classes to apply in the switch statement.

That said, your comment did want to make me double check that the alert that uses the `alert-error` class, is working as expected. And unfortunately it's not.

A quick fix could be to add `alert-error` to the switch statement, as indicated below.

```
    case 'alert-error':
      alertTypeStyle = 'bg-red-600 text-white alert-error'; <--- add alert-error here
      break;
```

Local
![image](https://user-images.githubusercontent.com/10620585/196779180-6be951b5-eefb-44c5-8f56-92138cab18a0.png)

Stage
![image](https://user-images.githubusercontent.com/10620585/196779256-b7bad742-8181-41a2-9dc6-beedc2401a98.png)

Reply:
To easily replicate this scenario, you can create a subscription using PayPal, and then deleting the billing agreement in Stripe.